packages/@uppy/transloadit/src/index.js

```diff
@@ -153,7 +153,9 @@ module.exports = class Transloadit extends Plugin {
     // Add Assembly-specific Tus endpoint.
     const tus = {
       ...file.tus,
-      endpoint: status.tus_url
+      endpoint: status.tus_url,
+      // Include X-Request-ID headers for better debugging.
+      addRequestId: true
```

Comment:
is there any reason we are not adding this to companion as well? 🤔 There are no CORS concerns there, so adding probably shouldn't be a problem, no?

**One question about the requestId;** Are the request Ids unique to each tus-js-client request, or are they unique to an upload instance instead?

Reply:
> is there any reason we are not adding this to companion as well? thinking There are no CORS concerns there, so adding probably shouldn't be a problem, no?

What do you mean with "adding to companion"? If you want to add the `X-Request-ID` header to your requests, the server's CORS respond must include this header in the list of allowed headers. So you cannot send it to an arbitrary (tus) server. That's also why this feature is disabled by default.

> Are the request Ids unique to each tus-js-client request, or are they unique to an upload instance instead?

They are unique to each request.

Reply:
> What do you mean with "adding to companion"? If you want to add the X-Request-ID header to your requests, the server's CORS respond must include this header in the list of allowed headers.

So I'm asking if there's a reason we are not also enabling the `addRequestId` option when using `tus-js-client` in companion [here](https://github.com/transloadit/uppy/blob/de235423c05e2803fecebafde92cfc2fdbdb6346/packages/%40uppy/companion/src/server/Uploader.js#L448-L483). Since companion is a server to server (companion <--> tusd) communication, we shouldn't have to worry about CORS here right?

Reply:
Good idea, I added the option :+1: